pkg/image/registry/imagestream/strategy.go

```diff
@@ -156,12 +156,6 @@ func (s Strategy) tagsChanged(old, stream *api.ImageStream) fielderrors.Validati
 		}
 
 		if tagRef.From.Kind == "DockerImage" && len(tagRef.From.Name) > 0 {
-			event, err := tagReferenceToTagEvent(stream, tagRef, "")
-			if err != nil {
-				errs = append(errs, fielderrors.NewFieldInvalid(fmt.Sprintf("spec.tags[%s].from", tag), tagRef.From, err.Error()))
-				continue
-			}
-			api.AddTagEventToImageStream(stream, tag, *event)
```

Comment:
No, it's not ok.  If you want to import something, we shouldn't create the status tag until it's imported.  If you don't want to import it, but instead to direct tag it, it shouldn't get a second tag.  If we want to import by default, then we need to bypass this.